packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -182,6 +182,42 @@ export function getStaticLivenessOvalDetails({
   };
 }
 
+export function drawStaticOval(
```

Comment:
Helper method for drawing a static oval. Used to redraw the static oval when the color mode changes since we need to re-compute the style for the background color